docs/core/bcc.rst

```diff
@@ -7,14 +7,14 @@ api-blockchain-core-browserified
    :stub-columns: 1
 
    * - Source
-     - `api-blockchain-core-browserified <https://github.com/evannetwork/ui-core/tree/master/dapps/bcc>`__
+     - `api-blockchain-core-browserified <https://github.com/evannetwork/ui-dapps/tree/master/dapps/bcc>`__
 
 
 evan.network framework
 ======================
-Using the browserified blockchain-core version within the dapp-browser, you need apply the following configurations to your dapp's project. Be sure, to exclude it from your build jobs. The angular-gulp project will handle it automatically. By using vue applications, it will be configured within the `vue/webpack.config.js <https://github.com/evannetwork/ui-core-dapps/blob/master/vue/webpack.externals.js>`__ file.
+Using the browserified blockchain-core version within the dapp-browser, you need apply the following configurations to your dapp's project. Be sure, to exclude it from your build jobs. The angular-gulp project will handle it automatically. By using vue applications, it will be configured within the `vue/webpack.config.js <https://github.com/evannetwork/ui-dapps/blob/master/script/dapp/webpack.externals.js>`__ file.
```

Comment:
```suggestion
Using the browserified blockchain-core version within the dapp-browser, you need apply the following configurations to your dapp's project. Be sure, to exclude it from your build jobs. The angular-gulp project will handle it automatically. By using vue applications, it will be configured within the `vue/webpack.config.js <https://github.com/evannetwork/ui-dapps/blob/master/scripts/dapp/webpack.externals.js>`__ file.
```